<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceStatusWidget</class>
 <widget class="QWidget" name="DeviceStatusWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>42</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>42</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>42</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">DeviceStatusWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget {
background-image: url(:/Leonis/resource/image/bk-device-status.png);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="status_color_label">
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_first" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="widget2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="name_label_first">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="states" stdset="0">
            <string notr="true">color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="time_label_first">
           <property name="minimumSize">
            <size>
             <width>46</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>46</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="states" stdset="0">
            <string notr="true">color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status_label_first">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="states" stdset="0">
         <string notr="true">color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_second" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_second">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="name_label_second">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="states" stdset="0">
            <string notr="true">color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="time_label_second">
           <property name="minimumSize">
            <size>
             <width>46</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>46</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="states" stdset="0">
            <string notr="true">color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status_label_second">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="states" stdset="0">
         <string notr="true">color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: url(:/Leonis/resource/image/bk-device-status-right.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
